algorithmia/metadata.csv

```diff
@@ -0,0 +1,2 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+duration_milliseconds,gauge,,millisecond,,Duration of algorithm run,0,algorithmia,
```

Comment:
As it says in https://github.com/DataDog/integrations-extras/pull/727/files#diff-7eac1194a680f15116e7155e23416f62R6 metrics emitted by this integration should be prefixed with `algorithmia.` to be correctly identified as belonging to this integration. Although here https://github.com/DataDog/integrations-extras/pull/727/files#diff-a46bf5e60a1167da9b93115a5fae52a5R55-R56 it says the prefix is `algorithmia-insights.`.
Whichever is the correct prefix it should be reflected both in the `metadata.csv` and the `manifest.json` files (and also it must be true that the metrics are coming with that prefix)

Reply:
Fixed. Made the prefix consistent between `metadata.csv`, `manifest.json`, and the README. Thanks!